docker/dev/build.sh

```diff
@@ -19,28 +19,27 @@ set -e
 # $SQLFLOWPATH, or $PWD.
 SQLFLOWPATH=${SQLFLOWPATH:=$PWD}
 cd $SQLFLOWPATH
-
-# The directory saving binaries
 SQLFLOW_BIN=$SQLFLOWPATH/build
+echo "Build $SQLFLOWPATH into $SQLFLOW_BIN ..."
 
-# Build sqlflowserver, sqlflow, and step into $SQLFLOW_BIN
+echo "Build sqlflowserver, sqlflow, and step into $SQLFLOW_BIN ..."
 go generate ./...
 GOBIN=$SQLFLOW_BIN go install ./...
 
-# Build Couler
+echo "Build $SQLFLOWPATH/python/couler into $SQLFLOW_BIN ..."
 cd $SQLFLOWPATH/python/couler
-python setup.py bdist_wheel --dist-dir $SQLFLOW_BIN
+python setup.py bdist_wheel -q --dist-dir $SQLFLOW_BIN > /dev/null
```

Comment:
Like `apt-get -q install`, `setup.py bdist_wheel` also need `> /dev/null` to make it quiet.